tools/code_review_tools/lando.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you move this file to https://github.com/mozilla/libmozdata instead of adding it here?
We might want to use it in other projects too.

Reply:
OK, I will move it to libmozdada